lib/integrations/Apple/AppleCalendarAdapter.ts

```diff
@@ -52,12 +50,12 @@ export class AppleCalendar implements CalendarApiAdapter {
     });
   }
 
-  convertDate(date: string): number[] {
+  convertDate(date: string): [number, number, number] {
     return dayjs(date)
       .utc()
       .toArray()
       .slice(0, 6)
-      .map((v, i) => (i === 1 ? v + 1 : v));
+      .map((v, i) => (i === 1 ? v + 1 : v)) as [number, number, number];
```

Comment:
toArray results in an array with 7 numbers (including ms, which we don't care about) - we slice to 6 - so this typing is invalid even though it validates. I reckon we should remove all explicit typehints. Dayjs returns toArray as a number[] value which is honestly good enough. (if you overwrite it with as, you lose the benefit of fixed length anyway)

```javascript
  convertDate(date: string) {
    return dayjs(date)
      .utc()
      .toArray()
      .slice(0, 6)
      .map((v, i) => (i === 1 ? v + 1 : v));
  }
  ```